README.md

```diff
@@ -1,29 +1,131 @@
-# node-saucelabs [![Build Status](https://travis-ci.org/saucelabs/node-saucelabs.svg?branch=master)](https://travis-ci.org/saucelabs/node-saucelabs)
+<p align="center">
+    <img src="./assets/trophy-bot.png" alt="Trophy bot" />
+</p>
 
-Wrapper around the Sauce Labs REST API for [Node.js](http://nodejs.org/).
-
-> **NOTE:**. 
```

Comment:
I agreed with Dan that we would keep this in the file, we can move it to the bottom, but I think we still need to keep this

Reply:
Well the package has been completely rewritten. I am fine having a sentence at the bottom like "Originally created by ...". It has nothing to do with package it was before.

Reply:
I'm fine with that, as long as we mention them

Reply:
add note at the bottom of the readme